packages/editor/src/plugins/image-gallery/components/fitting-modal.tsx

```diff
@@ -0,0 +1,26 @@
+import * as Dialog from '@radix-ui/react-dialog'
+import React, { ReactNode } from 'react'
+
+interface FittingModalProps {
+  children: ReactNode
+  open: boolean
+  onOpenChange: (open: boolean) => void
+}
+
+export function FittingModal({
+  children,
+  open,
+  onOpenChange,
+}: FittingModalProps) {
+  return (
+    <Dialog.Root open={open} onOpenChange={onOpenChange}>
+      <Dialog.Portal>
+        <Dialog.Title className="sr-only" />
```

Comment:
Same as here: https://github.com/serlo/frontend/pull/3986#discussion_r1705362937. It would be great to add an actual title for screen readers.

If you don't agree, I'd love to hear your arguments against it. If you do agree, it's weird that you added an empty title again.